samples/unit-tests/maps/map-navigation/demo.html

```diff
@@ -1,6 +1,7 @@
 <script src="http://cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.6/proj4.js"></script>
 <script src="https://code.highcharts.com/maps/highmaps.js"></script>
 <script src="https://code.highcharts.com/mapdata/countries/gb/gb-all.js"></script>
+<script src="https://code.highcharts.com/mapdata/custom/world.js"></script>
```

Comment:
I think we don't need to load those maps now, after using simplified topoJSON in `json-sources.js` file.

Reply:
True - removed.
Just to clarify, the simplified TopoJSON from `json-srouces.js` is used for automated karma testing while the scripts in `demo.html` are used for utils in-browser testing. None of them uses the maps in `.js` format because TopoJSON maps are used instead. Mapped locally to `json-soruces.js` (karma) or fetched from CDN (utils).